### PR TITLE
fix: Use updated tree-kill instead of vendor

### PR DESCRIPTION
**Pull request by robcresswell**
This PR updates the minimum version of `tree-kill` to `1.2.2`, and removes the vendored version. The upstream version now contains the fix, so there's no need for us to maintain a fork.

See below for the diff from `v1.2.1` to `v1.2.2`

https://github.com/pkrumins/node-tree-kill/compare/v1.2.1...v1.2.2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,16 +28,17 @@
   "devDependencies": {
     "@types/node": "6.14.6",
     "@types/sinon": "7.0.11",
+    "debug": "^4.1.1",
```
Comment (dkontorovskyy) [inline package.json]: This should be still in prod deps

File: tslint.json [hunk 2]
```diff
@@ -1,24 +1,23 @@
 {
```
Comment (dkontorovskyy) [inline tslint.json]: Not sure how this file is related to `tree-kill`

File: package.json (hunk 1 above)
Comment (robcresswell) [reply package.json]: Ah good catch!

File: tslint.json (hunk 2 above)
Comment (robcresswell) [reply tslint.json]: See line 21, I had to exclude the vendoring, and prettier formatted it during the process

**Pull request merged**